src/core/reopt_inputs.jl

```diff
@@ -72,6 +72,16 @@ struct REoptInputs <: AbstractInputs
     pbi_max_benefit::Dict{String, Any}  # (pbi_techs)
     pbi_max_kw::Dict{String, Any}  # (pbi_techs)
     pbi_benefit_per_kwh::Dict{String, Any}  # (pbi_techs)
+    boiler_efficiency::Dict{String, Float64}
+    ghp_options::UnitRange{Int64}  # Range of the number of GHP options
```

Comment:
I agree that consistency would be nice. However, the PV(s) name thing is really just appending an ascending integer to the "PV" name, and I kind of like the simplicity of the UnitRange better than the strings. I also think it's OK that they are different because the PVs are appended to the list of `techs`, while GHP is not handled like a `tech`.

Reply:
For now let's call this resolved then, thanks for clarifying.  I think this is something to pick up again when we think about other heating systems like the electric heater and heat pumps discussed in https://github.com/NREL/REopt.jl/issues/64 because I do think an appropriate future state involves merging GHP and these new options into the techs.